src/mono/sample/wasm/browser-profile/Wasm.BrowserProfile.Sample.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <WasmCopyAppZipToHelixTestDir Condition="'$(ArchiveTests)' == 'true'">true</WasmCopyAppZipToHelixTestDir>
-    <WasmProfilers>aot;</WasmProfilers>
+    <WasmProfilers>aot;browser;</WasmProfilers>
```

Comment:
I think this sample is not testing browser profiler, just AOT profiler.

Reply:
Why is it called `BrowserProfile` then?

Reply:
good point, maybe you are right